### PR TITLE
Sorting the program out

### PR DESCRIPTION
**Pull request by GavenB141**

### CODE REVIEW

**Review by CooperW12 — COMMENTED** [review]
Does this print the foods? I'm not seeing "printfood();" in the main string args.

File: MyClass.java [hunk 1]
```diff
@@ -1,74 +1,71 @@
...
+            trunkNumber = (1+2*times)/3 + 1;
+        }else{
+            trunkNumber = (1+2*times)/3;
+        }
+        
+        for(int i=0; i<times; i++){
+            row(times-i,1+2*i);
+        }
+        stump((1+2*times), times);
+	}
```
Comment (Team1710CTO) [inline MyClass.java:22]: you will need to insert the printfood(); function somewhere around here

**Review by Team1710CTO — CHANGES_REQUESTED** [review]

File: MyClass.java (hunk 1 above)
Comment (GavenB141) [reply MyClass.java:22]: it's added

**Review by CooperW12 — APPROVED** [review]

**Review by Team1710CTO — APPROVED** [review]

**Review by Team1710CTO — APPROVED** [review]

**Review by TheChar — APPROVED** [review]

**Review by Andrew-K961 — APPROVED** [review]
Less than 5 doesn't work but everything else is great!

**Pull request merged**